collections/concepts/what-are-applications.md

```diff
@@ -47,9 +47,33 @@ Each Application Type has different levels of permission available for the appli
 
 Your Application’s API Key is used to authenticate your systems to the Basis Theory platform. The API Key can be used to make authenticated requests to our API directly, using Hosted Elements, or using the Basis Theory SDK. These API Keys follow a format similar to `key_4qUtg83907AnDem90aJSzcN`.
 
+## Application Templates
 
+To get you started quickly, we provide a set of Application Templates that cover the most common use cases. Once you click on
+"Create Application", a modal will appear to let you select an Application Template depending on your needs. If you 
+have a particular use case or want to create an Application step by step, you can click the `Create Your Own` option. It
+is important to note that these templates will only pre-fill the fields in the Create Application page, and once created,
+your Application has no relationship with the template in any way.
+
+<img alt="Application Templates" src="/assets/images/concepts/application-templates.png">
+
+In this modal you can see we provide templates for the most common usages such as `Payments`, `Banking` and `PII Data`.
+However, we also have the `Starters` section which allows you to create your own Application step by step if you have 
+a custom use case or want to get started quickly with a `Full Access` Application. The last section is `Management`, which 
+provides a `Full Management Access` template to get all the managing permissions available.
```

Comment:
IMO this is way too verbose and we don't need to describe every detail of the UI.

```suggestion
This modal displays several templates to quickly achieve common use cases, such as `Payments`, `Banking`, and `PII Data`. The `Full Access` template can be useful for creating a test Application with every permission, but it should not be used for production Applications. 
```